BetterRechner/SimpleCalculator.vb

```diff
@@ -48,62 +49,49 @@ Module SimpleCalculator
-                Catch ex As Exception
-                    Console.WriteLine("This is broken now.")
-                End Try
-            ElseIf mathFunction = "*" Or mathFunction = "Multiply" Or mathFunction = "Multiplication" _
+
+                ElseIf mathFunction = "*" Or mathFunction = "Multiply" Or mathFunction = "Multiplication" _
                 Or mathFunction = "Product" Or mathFunction = "Mult" Or mathFunction = "1" Then
-                Try
+
                     Console.WriteLine(firstNumber & "*" & secondNumber & "=" & firstNumber * secondNumber)
-                Catch ex As InvalidCastException
-                    Console.WriteLine("Please Enter a Whole Number.")
-                Catch ex As Exception
-                    Console.WriteLine("This is broken now.")
-                End Try
-            ElseIf mathFunction = "-" Or mathFunction = "Subtract" Or mathFunction = "Minus" _
+
+                ElseIf mathFunction = "-" Or mathFunction = "Subtract" Or mathFunction = "Minus" _
                 Or mathFunction = "Difference" Or mathFunction = "Sub" Or mathFunction = "3" Then
-                Try
+
                     Console.WriteLine(firstNumber & "-" & secondNumber & "=" & firstNumber - secondNumber)
-                Catch ex As InvalidCastException
-                    Console.WriteLine("Please Enter a Whole Number.")
-                Catch ex As Exception
-                    Console.WriteLine("This is broken now.")
-                End Try
-            ElseIf mathFunction = "Quotient" Or mathFunction = "/" Or mathFunction = "Divide" _
+
+                ElseIf mathFunction = "Quotient" Or mathFunction = "/" Or mathFunction = "Divide" _
                 Or mathFunction = "Division" Or mathFunction = "Div" Or mathFunction = "4" Then
-                Try
+
                     Console.WriteLine(firstNumber & "/" & secondNumber & "=" & firstNumber / secondNumber)
-                Catch ex As InvalidCastException
-                    Console.WriteLine("Please Enter a Whole Number.")
-                Catch ex As Exception
-                    Console.WriteLine("This is broken now.")
-                End Try
-            ElseIf mathFunction = "Exponent" Or mathFunction = "^" Or mathFunction = "To the Power of" _
+
+                ElseIf mathFunction = "Exponent" Or mathFunction = "^" Or mathFunction = "To the Power of" _
                 Or mathFunction = "5" Then
-                Try
+
                     Console.WriteLine(firstNumber & "^" & secondNumber & "=" & firstNumber ^ secondNumber)
-                Catch ex As InvalidCastException
-                    Console.WriteLine("Please Enter a Whole Number.")
-                Catch ex As Exception
-                    Console.WriteLine("This is broken now.")
-                End Try
-            Else
-                Console.WriteLine("That is not an appropiate Response.")
-            End If
+
+                ElseIf mathFunction = "" Then
```

Comment:
If the user presses enter here with no input. the recalculate flag gets set to false.